frontend/src/assets/index.js

```diff
@@ -20,6 +20,8 @@ import verifiedAccountDrawing from './VerifiedAccountPage/ill-confirmation.svg';
 
```

Comment:
General comment: The page has a scroll bar on the bottom. It can also be moved a bit vertically. This page does not require any scrolling and it should be fixed.
![image](https://user-images.githubusercontent.com/31959982/117119561-ee544100-ad9a-11eb-8094-6a9b5c27b5b6.png)